src/main/spec/internal-billing-api-spec.yaml

```diff
@@ -19,13 +13,9 @@ servers:
           - stage.cloud.redhat.com
           - cloud.redhat.com
         default: ci.cloud.redhat.com
-      PATH_PREFIX:
-        default: api
-      APP_NAME:
-        default: rhsm-subscriptions
 
 paths:
-  /internal/remittance/accountRemittances:
+  /api/swatch-tally/internal/remittance/accountRemittances:
```

Comment:
I'm not sure if this is correct... should the billing API be mapped to the swatch tally?